File: ScaleformUI_Csharp/Menus/UIRadioMenu/RadioItem.cs
namespace ScaleformUI.Radio
{
    public class RadioItem
    {
        public UIRadioMenu Parent { get; internal set; }
        public string TextureDictionary { get; set; }
        public string TextureName { get; set; }
        public string StationName { get; set; }
        public string Artist { get; set; }
        public string Track { get; set; }

        public bool Selected { get; internal set; }

        public RadioItem(string station, string artist, string track, string txd, string txn)
        {
            TextureDictionary = txd;
            TextureName = txn;
            StationName = station;
            Artist = artist;
            Track = track;
        }
    }
}

File: ScaleformUI_Lua/src/Menus/RadioMenu/RadioItem.lua
RadioItem = setmetatable({}, RadioItem)
RadioItem.__index = RadioItem
RadioItem.__call = function()
    return "RadioItem"
end

function RadioItem.New(_station, _artist, _track, _txd, _txn)
    local data = {
        Parent = nil,
        StationName = _station,
        Artist = _artist,
        Track = _track,
        TextureDictionary = _txd,
        TextureName = _txn
    }
    return setmetatable(data, RadioItem)
end
